src/System.ServiceProcess.ServiceController/tests/System.ServiceProcess.ServiceController.Tests/System.ServiceProcess.ServiceController.Tests.csproj

```diff
@@ -30,6 +30,7 @@
       <Project>{ceb0775c-4273-4ac4-b50e-4492718051ae}</Project>
       <Name>System.ServiceProcess.ServiceController.TestNativeService</Name>
       <SetConfiguration>Configuration=$(ConfigurationGroup)</SetConfiguration>
+      <SetPlatform>Platform=x64</SetPlatform>
```

Comment:
The description says "on Nano", but this looks like "always".

Am I missing something?